togetherly/lib/views/widgets/new_chore_dialog.dart

```diff
@@ -9,31 +10,73 @@ import 'package:togetherly/themes.dart';
 import 'package:togetherly/utilities/date.dart';
 
 class NewChoreDialog extends StatefulWidget {
-  const NewChoreDialog({super.key, this.assignedChildId});
-  final int? assignedChildId;
+  const NewChoreDialog({super.key, this.assignedChildId, this.chore});
+  final int?
+      assignedChildId; // left as optional depending on if the chore is created on the child page or the parent page
+  final Chore?
+      chore; // left as optional depending on if the parent is creating a new chore or editing an existing one
 
   @override
   State<NewChoreDialog> createState() => _NewChoreDialogState();
 }
 
 class _NewChoreDialogState extends State<NewChoreDialog> {
-  static const List<String> _weekdayAbbreviations = ['S', 'M', 'T', 'W', 'Th', 'F', 'Sa'];
+  static const List<String> _weekdayAbbreviations = [
+    'S',
+    'M',
+    'T',
+    'W',
+    'Th',
+    'F',
+    'Sa'
+  ];
 
   final _formKey = GlobalKey<FormState>();
-  final _titleController = TextEditingController();
-  final _descriptionController = TextEditingController();
-  final _pointsController = TextEditingController();
+  final MultiSelectController<Child> selectController = MultiSelectController();
 
   DateTime _dueDate = DateTime.now();
-  String _title = '';
-  String _description = '';
-  int _points = 0;
-  bool _isBonus = false;
-  bool _isShared = false;
+  late String _title;
+  late String _description;
+  late int _points = 0;
+  late bool _isBonus;
+  late bool _isShared;
   bool _loading = false;
+  List<Child> _assignedPeople = [
+    const Child(
+      familyId: 0,
+      name: 'Emma',
+      icon: ProfileIcon.bear,
+      totalPoints: 45,
+    ),
+    const Child(
+      familyId: 0,
+      name: 'Jacob',
+      icon: ProfileIcon.dog,
+      totalPoints: 80,
+    ),
+  ];
+  late List<ValueItem<Child>> _assignedValueItems;
```

Comment:
Is there any reason this is part of the state, given it only seems to be used as a local variable in `initState`?

Reply:
Good point. I removed the state variable declaration and created a local variable in the initState